config/about.config.ts

```diff
@@ -52,6 +52,7 @@ export const aboutPageData: AboutPageData = {
       links: [
         { name: 'TIP', link: '/education/tip' },
         { name: 'mentor program', link: '/education/mentor' },
+        { name: 'learn more', link: '/education' },
```

Comment:
This will cause a 404 error if the user clicks the button bc we don't have a page here